pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -114,6 +121,19 @@ type ConfigCheck struct {
 	Labels map[string]string `json:"labels,omitempty"`
 }
 
+type RouteConfig struct {
+	// If DisableLoggingRoute is set to true, the logging route controller
+	// should remove the given tenant from the status of the logging resource.
+	DisableLoggingRoute bool `json:"disableLoggingRoute,omitempty"`
+	// If EnableTelemtryControllerRoute set to true, the operator will create
+	// the corresponding Tenant, Subscription, Output based on the logging resource.
+	EnableTelemetryControllerRoute bool `json:"enableTelemetryControllerRoute,omitempty"`
+	// TenantLabels is a map of labels that will be added to the tenant object
+	// so it can be matched with TelemetryController's TenantSelector
+	// ref: https://github.com/kube-logging/telemetry-controller/blob/main/api/telemetry/v1alpha1/collector_types.go
+	TenantLabels *metav1.LabelSelector `json:"tenantLabels,omitempty"`
```

Comment:
We might want to change this to a simple `map[string]string`
@OverOrion @pepov WDYT?